src/olympia/editors/models.py

```diff
@@ -126,22 +126,19 @@ def base_query(self):
             ]),
             'from': [
                 'addons',
-                """JOIN (
-                    SELECT MAX(id) AS latest_version, addon_id FROM versions
-                    GROUP BY addon_id
-                    ) AS latest_version
-                    ON latest_version.addon_id = addons.id
-                LEFT JOIN versions
-                    ON (latest_version.latest_version = versions.id)
-                JOIN files ON (files.version_id = versions.id)""",
+                """
+                LEFT JOIN versions ON (addons.id = versions.addon_id)
+                LEFT JOIN files ON (files.version_id = versions.id)
```

Comment:
my sql foo is weak but should this be a `JOIN` or versions with BETA files will be included?